tests/features/generator_expressions_pep289_test.py

```diff
@@ -20,6 +20,10 @@
             '"".join([[i for _ in range(2)] for i in range(3)])\n',
             id='string join (left alone for perf reasons)',
         ),
+        pytest.param(
+            'sum([i async for i in foo()])\n',
```

Comment:
this test doesn't fail before the change in python3.6 (this is a syntax error: `async for outside of async def`)